rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -22,7 +22,7 @@ servers:
   - url: ${rest.server.url}
 
 paths:
-  /product/{productId}/cst:
+  /product/{productId}:
```

Comment:
✏️ `/index/{productId}:`

Reply:
Wouldn't this make it less resource-based semantically speaking? What about `/product/{productId}/index`?

Reply:
The store API will still have a store metadata endpoint. That service is the one that needs to be "resource-based".